### ISSUE TITLE
firebase 10.24.0 crash 

### ISSUE THREAD

**Issue by iOSyifeng** [firebase/firebase-ios-sdk#12859]
### Description

[NSConcreteFileHandle writeData:]: No space left on device
__90-[FIRCLSRolloutsPersistenceManager updateRolloutsStateToPersistenceWithRollouts:reportID:]_block_invoke (FIRCLSRolloutsPersistenceManager.m:63)

### Firebase SDK Version

10.24.0

### Xcode Version

15.3

### Installation Method

CocoaPods

### Firebase Product(s)

Remote Config

### Targeted Platforms

iOS

### Relevant Log Output

```shell
[NSConcreteFileHandle writeData:]: No space left on device
__90-[FIRCLSRolloutsPersistenceManager updateRolloutsStateToPersistenceWithRollouts:reportID:]_block_invoke (FIRCLSRolloutsPersistenceManager.m:63)
```

### If using Swift Package Manager, the project's Package.resolved

[NSConcreteFileHandle writeData:]: No space left on device
__90-[FIRCLSRolloutsPersistenceManager updateRolloutsStateToPersistenceWithRollouts:reportID:]_block_invoke (FIRCLSRolloutsPersistenceManager.m:63)

### If using CocoaPods, the project's Podfile.lock

[NSConcreteFileHandle writeData:]: No space left on device
__90-[FIRCLSRolloutsPersistenceManager updateRolloutsStateToPersistenceWithRollouts:reportID:]_block_invoke (FIRCLSRolloutsPersistenceManager.m:63)

**Comment by paulb777** [comment]
Based on the error message, the device was out of space. We might be able to handle it more gracefully, but it's unlikely the app will be able to succeed.

**Comment by paulb777** [comment]
The fix was merged from #12863 and will be in the 10.27.0 release.
